src/SldStyleParser.ts

```diff
@@ -95,6 +98,52 @@ const COMBINATION_MAP = {
 
 type CombinationType = keyof typeof COMBINATION_MAP;
 
+const SUPPORTED_LOCALES = ['en', 'de', 'fr'] as const;
+type Locale = (typeof SUPPORTED_LOCALES)[number];
+
+export type SldStyleParserTranslationKeys = {
+  marksymbolizerParseFailedUnknownWellknownName?: string;
+  noFilterDetected?: string;
+  symbolizerKindParseFailed?: string;
+  colorMapEntriesParseFailedColorUndefined?: string;
+  contrastEnhancParseFailedHistoAndNormalizeMutuallyExclusive?: string;
+  channelSelectionParseFailedRGBAndGrayscaleMutuallyExclusive?: string;
+  channelSelectionParseFailedRGBChannelsUndefined?: string;
+};
+
+export type SldStyleParserTranslations = Record<Locale, Partial<SldStyleParserTranslationKeys>>;
```

Comment:
See comment above regarding limiting translations.

```suggestion
export type SldStyleParserTranslations = Record<string, Partial<SldStyleParserTranslationKeys>>;
```